unity-client/Assets/Scripts/MainScripts/DCL/Environment/Environment.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.ComponentModel;
+using DCL.Controllers;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace DCL
+{
+    public class Environment
+    {
+        public static Model i = new Model();
```

Comment:
I find a bit confusing that `Environment.i` wont return an `Environment` instance. I have no alternative though.

Reply:
In order to avoid the confussion, maybe in this case we should not call this variable as `i` (we always associate `i` as "instance") and rename it by something like `model` or something like that. What do you think?

Reply:
We can rename it to `model`, but it will add considerable syntax overhead and that's rather undesirable. Let me think about this. In any case we can agree to make this change in another PR? Because it will add a lot of changes and break the subsequent PRs.

Reply:
Sure thing, I approved it already taking that into account

Reply:
Of course, if it will provoke a lo of syntax overhead, we can do it on another PR in the near future 👍